src/app/plugins/pubsub.plugin.ts

```diff
@@ -38,84 +40,66 @@ export class PubSubPlugin extends Plugin {
-
-    await this._updateList();
-
-    //setting up number of columns and rows for the list
-    const maxCol = 3;
-    const maxRows = 10;
-    const numCols = Math.min(maxCol, Math.ceil(this._SUBS.length / maxRows));
-    const numRows = Math.ceil(this._SUBS.length / numCols);
+    this._LAST_UPD_TIME = Date.now(); // since it only updates once a day, don't need to worry about accuracy
 
-    //sets up a new embed for the list to reside in
-    this._EMBED_LIST = new MessageEmbed();
-    this._EMBED_LIST.setColor('#0099ff').setTitle('Types of Subs');
-    const subs = this._SUBS.map((sub) => this._normalizeName(sub));
+    await this._updateData();
 
-    //cycles through each column inserting their row, and also notes the alphabetic range
-    for (let cols = 0; cols < numCols; cols++) {
-      const columnSubs = subs.slice(numRows * cols, numRows * (cols + 1));
+    this._EMBED_LIST.addField('Available Subs', this._SUBS.join('\n'));
 
-      //each column is a embed field note: only 3 fields can be lined up
-      this._EMBED_LIST.addField(
-        `${columnSubs[0].charAt(0)} - ${columnSubs[columnSubs.length - 1].charAt(0)}`,
-        columnSubs.join('\n'),
-        true
-      );
-    }
+    return this._EMBED_LIST;
   }
 
-  private _generateEmbedSub(subData: any): MessageEmbed {
+  private _generateEmbedSub(subData: subData): MessageEmbed {
     const embed: MessageEmbed = new MessageEmbed();
-    const parsedNamed = this._normalizeName(subData.sub_name);
 
-    //defaults to not on sale
-    let saleInfo = `it **isn't** on sale currently :sob:, but who could say no to ${subData.price}.  The last time it was on sale was ${subData.last_sale}`;
+    // defaults to not on sale
+    let saleInfo =
+      `it **isn't** on sale currently 😭, but who could say no to ${subData.price}.` +
+      ` The last time it was on sale was ${subData.last_sale}`;
 
-    //status is true if the sub is on sale
-    if (subData.status.toLowerCase() === 'true') {
-      saleInfo = `it **is** on sale for ${subData.price} :partying_face:, until ${
-        subData.last_sale.split('-')[1]
-      }`;
+    // status is true if the sub is on sale
+    if (subData.status) {
+      saleInfo = `it **is** on sale for ${subData.price} 🥳, until ${subData.last_sale_end}`;
     }
 
-    embed.setColor('#0099ff').setTitle(parsedNamed + ' sub');
+    embed.setColor('#0099ff').setTitle(subData.sub_name + ' sub');
 
     embed.setDescription(
-      `Get your own *${parsedNamed}* sub, ${saleInfo}.` +
+      `Get your own *${subData.sub_name}* sub, ${saleInfo}.` +
         ` So what are you waiting for?  Come on down to Publix now to get yourself a beautiful sub.` +
-        ` Just look at this beauty right here! 😍😍😍:pubsub::pubsub::pubsub:🤤🤤🤤`
+        ` Just look at this beauty right here! ` +
+        `😍😍😍${this._PUB_SUB_EMOJI || '🥪'}${this._PUB_SUB_EMOJI || '🥪'}${this._PUB_SUB_EMOJI ||
```

Comment:
instead of `|| 'sandwich'` here you should do it when you initially assign `PUB_SUB_EMOJI` earlier